### PR TITLE
v0.3 Release 

### PR DESCRIPTION
**Pull request by OlliePorter**
## Description

This release adds the Transaction History (Tx History) feature and fixes some UI bugs from `v0.2`. A full change breakdown of `v0.3` includes:
- New TX History page which displays an empty state message when no transactions are present
- When transactions are present, it will display each transaction in a table card with each data formatted in a row
- Transaction row will show `+` amount and `received` text when transaction was received to the current wallet address
- Transaction row will show `-`  amount and `sent` text when transaction was sent from the current wallet address to another address
- Transaction row will show correct `+/-` amount based on sent/received logic
- Transaction row will show `Confirmed` in a `StatusChip` widget when the transaction is at a block depth of 30 or greater (defaults to `Unconfirmed`)
- Transactions can be filtered by all, sent or received in the dropdown (minted transactions will only appear in all, not in sent or received) 
- Tapping/clicking on a transaction row will redirect to the transaction details page with a full table breakdown of the transaction data
- `RibnBottomAppBar` now has a new look with the plus icon (for minting journey) now inside a green bubble
- The `Accordion `widget in onboarding now has the appropriate spacing
- Double buttons in `BottomReviewAction` are no longer cutoff in Chrome
- Text has been made smaller on Login page for Chrome 
- `CustomToolTip` now has a grey border with default white background for better contrast on lighter pages

194370940-443c4a5b-9ecb-450f-bacb-d12075438685 Screen Shot 2022-10-06 at 11 33 44 AM
 
## Type of change
-   [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
-   [X] Has been tested locally with mock data formatted using the correct Genus models (but not real data)
-   [x] Has yet to be tested locally on a private node with Genus installed for real data

### CODE REVIEW

File: lib/genus/protobuf/package.proto [hunk 1]
```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+import "scalapb/scalapb.proto";
+
+package co.topl.genus;
+
+option (scalapb.options) = {
+  scope: PACKAGE
+  flat_package: false
+};
```
Comment (OlliePorter) [inline lib/genus/protobuf/package.proto:10]: Will add the newline

File: lib/models/ribn_network.dart [hunk 2]
```diff
@@ -6,6 +6,7 @@ import 'package:flutter/foundation.dart';
 import 'package:ribn/constants/network_utils.dart';
 import 'package:ribn/constants/rules.dart';
 import 'package:ribn/models/ribn_address.dart';
+// import 'package:ribn/platform/platform.dart';
```
Comment (OlliePorter) [inline lib/models/ribn_network.dart:9]: Will remove commented import

File: lib/presentation/transaction_history/transaction_history_details_page/transaction_history_details_page.dart [hunk 3]
```diff
@@ -0,0 +1,469 @@
+// ignore_for_file: unused_import, unused_element
```
Comment (OlliePorter) [inline lib/presentation/transaction_history/transaction_history_details_page/transaction_history_details_page.dart:1]: Will remove

File: lib/presentation/transaction_history/transaction_history_page.dart [hunk 4]
```diff
@@ -0,0 +1,223 @@
+// import 'dart:convert';
```
Comment (OlliePorter) [inline lib/presentation/transaction_history/transaction_history_page.dart:1]: Will remove commenteded out imports

**Review by scasplte2 — APPROVED** [review]
🔥

**Pull request merged**